src/components/activity-card/activity-card.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import {
+  DashboardCard,
+  DashboardCardBody,
+  DashboardCardHeader,
+  DashboardCardTitle,
+  ActivityBody,
+  RecentEventsBody,
+} from '@console/dynamic-plugin-sdk/provisional';
+import {
+  useK8sWatchResource,
+} from "@console/dynamic-plugin-sdk/api";
+import { FirehoseResource } from "@console/dynamic-plugin-sdk";
+import {getNamespace} from "../../selectors/index";
+//import { IBM_STORAGE_ODF_OPERATOR } from '../../constants/index';
+import {
+   PersistentVolumeClaimModel, 
+   EventModel,    
+  } from '../../models';
+import './activity-card.scss';
+import {GetFlashSystemResource} from '../../constants/resources';
+import { StorageInstanceKind, EventKind } from '../../types';
+
+const eventsResource: FirehoseResource = { isList: true, kind: EventModel.kind, prop: 'events' };
+
+const RecentEvent = (props) =>{
+  const [data, loaded, loadError] = useK8sWatchResource<StorageInstanceKind>(GetFlashSystemResource(props?.match?.params?.name, props?.match?.params?.namespace));
+  const namespace= loaded && !loadError? data?.[0]?.metadata.namespace: '';
+  const odfEventNamespaceKindFilter = (event: EventKind): boolean => {
+    const eventKind = event?.involvedObject?.kind;
+    const eventNamespace = getNamespace(event);
+    return eventNamespace === namespace || eventKind === PersistentVolumeClaimModel.kind;
```

Comment:
This will show PVC in any namespace and of any provisioner in cluster.

Reply:
I find it is hard for me to filter out the exact provisioner PVC. Any suggestion? If we cannot, I think we can remove this.

Reply:
Yes, we had this same issue in our ocs dashboards. In later releases when ceph-csi supported snapshot it started annotating ceph based PVCs by `volume.beta.kubernetes.io/storage-provisioner` so we  were able to filter some of the events.( This solution do not covers Pending PVC though)


We can either omit it or keep it (depends on if we are okay with non flash pvc showing). Its up to you,